### ISSUE TITLE
Filename length is limited even after long path support enabled

### ISSUE THREAD

**Issue by xylosper** [conan-io/conan#1662]
I keep see next messages during 'Copying sources to build folder' stage:

```
WARN: Filename too long, file excluded: <some long file path>
```

I've already enabled long path support on my Windows 10.
However, as far as I can see, it seems that the length of file path is hard-coded here: https://github.com/conan-io/conan/blob/073b17c0bb548af6ddf6511c87e97a269814b6ba/conans/client/installer.py#L456

I know I can set `short_paths` attribute but I don't want to use the option because `short_paths` option pollutes outside of data folder.

Is this for python's limitation or can be fixed in conan?

**Comment by memsharded** [comment]
No, I think it is some legacy check, prior to Windows 10 enabling long paths.
Yes, this has to be improved, right now I am not sure of an approach that could be valid for both users with path limits and those who not. But lets try for 0.27

**Comment by memsharded** [comment]
Setting CONAN_USER_HOME_SHORT env-var to "None", or using ``[general] user_home_short = None`` in conan.conf will disable short_paths feature completely. Merged to develop, will be released in 0.27

**Comment by memsharded** [comment]
Released in 0.27, please update and try
